plugin-enclave-gradle/src/main/kotlin/com/r3/conclave/plugin/enclave/gradle/LinuxExec.kt

```diff
@@ -119,9 +113,24 @@ open class LinuxExec @Inject constructor(objects: ObjectFactory) : ConclaveTask(
         return null
     }
 
+    /** Returns the output of the command executed in the container. */
+    fun execWithOutput(params: List<String>): String {
+        val args: List<String> = listOf(
+            "docker",
+            "run",
+            "-i",
+            "--rm",
+            "-v",
+            "${baseDirectory.get()}:/project",
+            tag.get()
+        ) + params.map { it.replace(baseDirectory.get(), "/project").replace("\\", "/") }
+
+        return commandWithOutput(*args.toTypedArray()).trimEnd()
+    }
+
     fun throwOutOfMemoryException(): Nothing = throw GradleException(
-        "The sub-process ran out of RAM. On macOS or Windows, open the Docker preferences and " +
-        "alter the amount of memory granted to the underlying virtual machine. We recommend at least 6 gigabytes of RAM " +
-        "as the native image build process is memory intensive."
+        "The sub-process ran out of RAM. Open the Docker preferences and " +
```

Comment:
I don't think this issue is relevant for Docker on Linux since it runs natively. For Mac and Windows a Linux VM is required, hence the need to make sure the VM has enough memory allocated to it.

Reply:
Done.